### PR TITLE
website: Fix dead link for project example

### PR DESCRIPTION
**Pull request by jwflory**
Since the first edition of this page, the projects section was
rearranged significantly. This updates the example in the Runbook and
links to an updated commit for an example in the repo.

### CODE REVIEW

File: docs/infra/website.md [hunk 1]
```diff
@@ -220,23 +220,31 @@ Additionally, the `permalink` property is commonly used to give a more special U
 
 #### Example project
 
-See [2018-01-01-teleirc.md](https://github.com/FOSSRIT/fossrit.github.io/blob/67282a55a6330a5c61397eaeac92ef9c025f0432/projects/_posts/2018-01-01-teleirc.md):
+See [2020-03-01-teleirc.md](https://github.com/FOSSRIT/fossrit.github.io/blob/6d11475b205bbb564cb5b284b390b4810bec5d9b/projects/_posts/2020-03-01-teleirc.md):
```
Comment (Nolski) [inline docs/infra/website.md]: Why point to the md blob instead of the actual page?

**Comment by ritjoe** [comment]
On Wed, Mar 11, 2020 at 07:51:46AM -0700, Mike Nolan wrote:
> @Nolski approved this pull request.
> 
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> 
> In docs/infra/website.md:
> 
> > @@ -220,23 +220,31 @@ Additionally, the `permalink` property is commonly used to give a more special U
> 
>  #### Example project
> 
> -See [2018-01-01-teleirc.md](https://github.com/FOSSRIT/fossrit.github.io/blob/67282a55a6330a5c61397eaeac92ef9c025f0432/projects/_posts/2018-01-01-teleirc.md):
> +See [2020-03-01-teleirc.md](https://github.com/FOSSRIT/fossrit.github.io/blob/6d11475b205bbb564cb5b284b390b4810bec5d9b/projects/_posts/2020-03-01-teleirc.md):
> 
> Why point to the md blob instead of the actual page?
>

Not sure, but maybe because the context is about editing the source document rather than its rendered counterpart?

I have to say, this is the sort of thing, documentation about documentation, in which I easily find myself lost in other projects. It becomes a problem eg when one way of rendering the document (GitHub's in-place rendering) differs from how the document appears elsewhere (eg, at some other not-necessarily Github-hosted URL). One finds the document accumulating fixes for one form that then break the other, round and round.

File: docs/infra/website.md (hunk 1 above)
Comment (jwflory) [reply docs/infra/website.md]: @Nolski @ritjoe The context is for someone who is creating and adding new projects to the website, where they will need the raw Markdown format to create new projects. I presume if someone wants to find an example of a rendered project, they can browse the website itself. :slightly_smiling_face:

**Comment by jwflory** [comment]
Going to merge this through, thanks for the review @Nolski. :clapper:

**Pull request merged**